bindata/network-diagnostics/network-check-target.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Is there's a reason this is on every node? Wouldn't a deployment be better?

Reply:
Because we want to do a check from a pod to every node to cover the whole cluster.

Reply:
The rationale we agreed was:

From a pod to a pod on every node
From a pod to every kubeapiserver service/endpoint
From a pod to every openshiftapiserver service/endpoint
From a pod to LBs

Reply:
gotcha.